### DISCUSSION TITLE
Trigger GitHub actions' workflow when some workflows themselves change

### DISCUSSION THREAD

**Discussion by pantelis-karamolegkos** [community/community#32283] [Actions] [answered]
Ι want to create a workflow that is triggered when some **other** `.github/workflow` files change.

I want to monitor all the `.github/workflow` starting with a specific prefix. i.e. `foo_`, as in `.github/workflow/foo_1.yaml` , `.github/workflow/foo_2.yaml` etc.

Is it possible to specify such a pattern on the workflow trigger?

**Comment by airtower-luna** [comment, marked as answer]
Sort of. For events that support it (`push`, `pull_request`) you can [apply a path filter](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) as usual. There's no special treatment of workflow files in that regard.
